src/diffusers/loaders.py

```diff
@@ -1151,6 +1218,13 @@ def _modify_text_encoder(cls, text_encoder, lora_scale=1, network_alpha=None, ra
             )
             lora_parameters.extend(attn_module.out_proj.lora_linear_layer.parameters())
 
+        for _, mlp_module in text_encoder_mlp_modules(text_encoder):
+            mlp_module.fc1 = PatchedLoraProjection(mlp_module.fc1, lora_scale, network_alpha, rank=rank, dtype=dtype)
+            lora_parameters.extend(mlp_module.fc1.lora_linear_layer.parameters())
+
+            mlp_module.fc2 = PatchedLoraProjection(mlp_module.fc2, lora_scale, network_alpha, rank=rank, dtype=dtype)
+            lora_parameters.extend(mlp_module.fc2.lora_linear_layer.parameters())
+
```

Comment:
This will _always_ patch the mlp layers. I think this behavior should be opt-in, so we can choose to patch these layers when loading the lora checkpoints that this PR enables. I believe this will break loading existing lora checkpoints because those checkpoints do not have weights for the linear layers. (I do believe the weights are initialized to zero so it wouldn't change output values but regardless I would consider extraneous weights that we don't intend to use breakage)